hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/AbstractYarnScheduler.java

```diff
@@ -724,9 +724,7 @@ protected void autoCorrectContainerAllocation(List<ResourceRequest> resourceRequ
           if (extraContainers > 0) {
             // Change the state of the container from ALLOCATED to EXPIRED since it is not required.
```

Comment:
Please consider editing this comment since the container will be released instead of expired.

Reply:
Done. Thanks @shameersss1 for reminding this!